torch/_export/pass_base.py

```diff
@@ -264,12 +259,11 @@ def _fx(
         args: Tuple[Argument, ...],
         kwargs: Dict[str, Argument],
         meta: NodeMetadata,
-        interpreter: torch.fx.Interpreter,
     ) -> ProxyValue:
         args_data, kwargs_data = pytree.tree_map_only(
             ProxyValue, lambda x: x.data, (args, kwargs)
         )
-        res_data = getattr(interpreter, kind)(target, args_data, kwargs_data)
+        res_data = getattr(InlineInterpreter(), kind)(target, args_data, kwargs_data)
```

Comment:
Previously it seems like we used to reuse interpreter, but now we are creating a new one each time we invoke fx operator. Is it intended?

Reply:
You're right -- sorry, I don't think we need the InlineInterpreter anymore...will just get rid of it and use the interpreter that we already have 😂